### ISSUE TITLE
Is AutoFixture dead?

### ISSUE THREAD

**Issue by Romfos** [AutoFixture/AutoFixture#1297]
Hi,

Is AutoFixture dead?
As I see the latest commit in v5 was more than year ago

Thank you

**Comment by aivascu** [comment]
Hello @Romfos 

TL;DR No it is not dead.

There is currently work being done outside v5, and I am maintaining an active presence on StackOverflow and in GitHub Discussions, for anyone who needs help.
 
There were some issues with configuring the CI for the v5 branch. Those issues remained unresolved till last month when I have found a way to unblock things.

One issue was with making versioning work based on the branch name. I tried (many) different implementations, none of which worked properly. In the end I decided to move to GitVersion. In CD mode it supports versioning using release branches. So expect the `v5` branch to be renamed to `release/5.0.0` in the following days.

Another issue is AppVeyor throwing a null reference exception when uploading test results files without records. This issue appeared after moving the build script to NUKE. I have been investigating this for a long time. The solution I believe will be to ignore the error since it doesn't affect anyway the number of tests being displayed in the summary.

In the near future I think the project will move away from AppVeyor. Likely the replacement will be GitHub Actions.

Admittedly I have not been as active as I would have liked. That is due to my job and personal life, taking most of my time. It is really hard to find motivation, to do anything, after a day dealing with legacy code, and a dozen meetings.

Please keep in mind that AutoFixture is a community driven project, and we rely on contributions from the community to keep growing. Leave a star on the repository, so this project can find more contributors/maintainers.
